days/day-06/solutions/day06.stektpotet.go

```diff
@@ -14,8 +14,8 @@ func onNewBatch(data []byte, atEOF bool) (advance int, token []byte, err error)
   }
   // iteratively find instances of double new line, and advance to the next batch
   // (skip 4 bytes ahead past double newline)
-  if i := bytes.Index(data, []byte{13, 10, 13, 10}); i >= 0 {
-    return i + 4, data[0:i], nil
+  if i := bytes.Index(data, []byte{10, 10}); i >= 0 {
+    return i + 2, data[0:i], nil
```

Comment:
Haha, what=? A lot of magic going on here :laughing:

Reply:
_carriage return fuckery_

This solution is dangerously fragile 😨

Reply:
Good we have tests then, in case we are moving to a platform where carrige return has different semantics, in the future :laugh